test/modules/packages/collections/CollectionNQueens.chpl

```diff
@@ -156,9 +156,10 @@ coforall loc in Locales do on loc {
   coforall tid in 0 .. #here.maxTaskPar {
     barrier.barrier();
     var nSpins : int;
+    var (exists, myBoard) : (bool, boardType);
     while found.read() < totalSolutions {
       // BUG: If we do not specify the type here, we get a compiler internal error...

```

Comment:
move this comment up with the declaration?

Reply:
Done.